<ui version="4.0" >
 <class>MainWidgetClass</class>
 <widget class="QDockWidget" name="MainWidgetClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>316</height>
   </rect>
  </property>
  <widget class="QWidget" name="widget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" colspan="5" >
     <widget class="QSplitter" name="splitter_2" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget" >
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QLabel" name="lbMessage" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoFillBackground" >
            <bool>true</bool>
           </property>
           <property name="frameShape" >
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow" >
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="alignment" >
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QLabel" name="lbLotName" >
             <property name="frameShape" >
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text" >
              <string>LOT.No</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leLotName" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>356</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QLabel" name="lbProductCount" >
             <property name="frameShape" >
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text" >
              <string>生産数</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leProductCount" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>356</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <item>
            <widget class="QLabel" name="lbLotCount" >
             <property name="frameShape" >
              <enum>QFrame::Panel</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Sunken</enum>
             </property>
             <property name="text" >
              <string>LOT数</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leLotCount" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
             <property name="readOnly" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QTreeWidget" name="trMasterList" >
        <column>
         <property name="text" >
          <string>1</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QTableWidget" name="twMasterList" />
     </widget>
    </item>
    <item row="1" column="0" >
     <widget class="QCheckBox" name="chNGPoint" >
      <property name="text" >
       <string>NG &amp;Mark Enable</string>
      </property>
      <property name="checked" >
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" >
     <widget class="QCheckBox" name="chShowPieceLine" >
      <property name="text" >
       <string>Show Piece Line</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" >
     <widget class="PasswordPushButton" name="pbSearch" >
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>31</height>
       </size>
      </property>
      <property name="text" >
       <string>Search</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3" >
     <spacer>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0" >
       <size>
        <width>251</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4" >
     <widget class="PasswordPushButton" name="pbMasterSelect" >
      <property name="minimumSize" >
       <size>
        <width>0</width>
        <height>31</height>
       </size>
      </property>
      <property name="text" >
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>PasswordPushButton</class>
   <extends>QPushButton</extends>
   <header>passwordpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
